website/src/pages/grading/grade-output.tsx

```diff
@@ -0,0 +1,231 @@
+
+      
+
+    </div>
+  </>;
+}
+
+function AnnotationCheckboxLi(props: { option: annotationBool}): JSX.Element {
+  let AdditionalExplanation = null;
+  if (props.option.additionalExplanation){
+    AdditionalExplanation = (
+      <a href ="#"
+      className="group flex items-center space-x-2.5 text-sm "
+      >
+        <QuestionMarkCircleIcon
+          className="h-5 w-5 ml-3 text-gray-400 group-hover:text-gray-500"
+          aria-hidden="true"
+        />
+      </a>
+    );
+  }
+  
+  return <>
+    <li className="form-check flex mb-1">
+      <input 
+        className="form-check-input appearance-none h-4 w-4 border border-gray-300 rounded-sm bg-white checked:bg-blue-600 checked:border-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
+        type="checkbox"
+        value=""
+        id={props.option.attributeName}
+      />
+      <label
+        className="flex ml-1 form-check-label  hover:cursor-pointer"
+        htmlFor={props.option.attributeName}
+      >
+        <span className='text-gray-800 hover:text-blue-700'>
+          {props.option.labelText}
+        </span>
+        {AdditionalExplanation}
+      </label>
+    </li>
+    </>;
+
+
+}
+
+
+interface annotationBool {
+  attributeName: string;
+  labelText: string;
+  additionalExplanation?: string;
+}
+
+const ANNOTATION_FLAGS: annotationBool[] = [
+// For the time being this list is configured on the FE.
+// In the future it may be provided by the API.
+  { attributeName: 'fails_task',
+    labelText: 'Fails to follow the correct instruction / task',
+    additionalExplanation: '__TODO__'
+  },
+  { attributeName: 'not_customer_assistant_appropriate',
+    labelText: 'Inappropriate for customer assistant',
+    additionalExplanation: '__TODO__'
+  },
+  { attributeName: 'contains_sexual_content',
+    labelText: 'Contains sexual content',
+  },
+  { attributeName: 'contains_violent_content',
+    labelText: 'Contains violent content',
+  },
+  { attributeName: 'encourages_violence',
+    labelText: 'Encourages or fails to discourage violence/abuse/terrorism/self-harm',
+  },
+  { attributeName: 'denigrates_a_protected_class',
+    labelText: 'Denigrates a protected class',
+  },
+  { attributeName: 'gives_harmful_advice',
+    labelText: 'Fails to follow the correct instruction / task',
+    additionalExplanation: 'The advice given in the output is harmful or counter-productive. This may be in addition to, but is distinct from the question about encouraging violence/abuse/terrorism/self-harm.',
+  },
+  { attributeName: 'expresses_moral_judgement',
+  labelText: 'Expresses moral judgement',
+  },
+]
+
+const SAMPLE_PROMPT = "Please make a list of aspects of a good pull request. Briefly describe each aspect.";
+
+const SAMPLE_OUTPUT = `Here are some aspects of a good pull request, which you may use to help your pull requests be good contributions and get accepted:
+
```

Comment:
hehe, I like it...

Reply:
ChatGPT has some nice output for this prompt of course but I didn't want to commit their output so I was handwriting this. It's an outline I was going to fill out more, but I ended up leaving it like this because it gets the basic point.

It's amazing how much time and effort is created to write a little essay about a subject like this; When you compare it to the near instant output of chatGPT it's flabbergasting. LLMs are thousands of times faster than us 😅